compiler/il/OMROpcodes.enum

```diff
@@ -7203,6 +7203,22 @@ OPCODE_MACRO(\
    /* .ifCompareOpCode      = */ TR::BadILOp, \
    /* .description          =    Inline code for memory comparison of part of an array */ \
 )
+OPCODE_MACRO(\
+   /* .opcode               = */ arraycmplen, \
+   /* .name                 = */ "arraycmplen", \
+   /* .properties1          = */ ILProp1::Call | ILProp1::HasSymbolRef, \
+   /* .properties2          = */ ILProp2::CondCodeComputation, \
```

Comment:
Looking at the other places where this property is used, does it apply to arrayCmpLen?